README.md

```diff
@@ -482,6 +482,7 @@ The `asdf` core provides a [security policy](https://github.com/asdf-vm/asdf/sec
 | PHP                           | [asdf-community/asdf-php](https://github.com/asdf-community/asdf-php)                                           | [![Build status](https://github.com/asdf-community/asdf-php/workflows/Main%20workflow/badge.svg?branch=master)](https://github.com/asdf-community/asdf-php/actions)                                                               |
 | pint                          | [sam-burrell/asdf-pint](https://github.com/sam-burrell/asdf-pint)                                               | [![Build Status](https://github.com/sam-burrell/asdf-pint/workflows/ci/badge.svg?branch=main)](https://github.com/sam-burrell/asdf-pint/actions/workflows/ci.yml)                                                                 |
 | pipectl                       | [pipe-cd/asdf-pipectl](https://github.com/pipe-cd/asdf-pipectl)                                                 | [![Build Status](https://github.com/pipe-cd/asdf-pipectl/actions/workflows/build.yml/badge.svg)](https://github.com/pipe-cd/asdf-pipectl/actions)                                                                                 |
+| pipx                          | [joe733/asdf-pipx](https://github.com/joe733/asdf-pipx)                                                         | [![Build Status](https://github.com/pipe-cd/asdf-pipectl/actions/workflows/build.yml/badge.svg)](https://github.com/pipe-cd/asdf-pipectl/actions)                                                                                 |
```

Comment:
The badges need to be updated to point to your plugins repo

Reply:
Fixed.